charts/tiphys/templates/app.yaml

```diff
@@ -172,5 +172,7 @@ spec:
               envFrom:
                 - secretRef:
                     name: {{ template "opszero.fullname" $ }}
+              resources:
```

Comment:
Check that resources exists first. Otherwise this will fail if no resources are passed.

{